lib/broccoli/default-packager.js

```diff
@@ -844,7 +845,9 @@ module.exports = class DefaultPackager {
         annotation: 'Tests To Process',
       });
 
-      if (this.isModuleUnificationEnabled) {
+      // src dir not there when using classic addon with MU dummy app
+      const srcPath = typeof tree === 'string' ? `${tree}/src` : 'src';
```

Comment:
I commented this case at https://github.com/ember-cli/ember-cli/pull/8322/files#r254987783

I think my implementation for skipping this logic provides a little more information about what/when and why this is happening. This issue requires further investigation, but I don't want to be a blocker atm.

Please, I would like you to copy my same implementation using the `dirNotFound` variable, and extend the doc comment if possible. I will continue the investigation of this particular case with `rwjblue`.